assessments/KRAssessment/output/academicSubjectDescriptors.jsonl

```diff
@@ -0,0 +1,5 @@
+{ "codeValue": "SocialFoundations", "description": "Social Foundations", "namespace": "uri://ed.sc.gov/AcademicSubjectDescriptor", "shortDescription": "Social Foundations" }
```

Comment:
you will want to add a gitignore to your output folder and ignore any files that end in jsonl so you avoid pushing the output